common/buildings/00_tradepost.txt

```diff
@@ -463,3 +463,301 @@ trade_post = {
+			always = yes
+		}
+		prerequisites = { wc_bank }		
+		gold_cost = 650
+		build_time = 650
+		army_reinforce_rate = 0.1
+		retinuesize = 100
+		retinue_maintenence_cost = -0.2
+		ai_creation_factor = 100
+	}
+	
+	#GoblinGhetto
+	wc_goblinGhetto = {
+		desc = wc_goblinGhetto_desc
+		potential = {
+			always = yes
+			NOT = { owner = { trait = creature_human } }
+			NOT = { owner = { trait = creature_high_elf } }
+			NOT = { owner = { trait = creature_blood_elf } }
+			NOT = { owner = { trait = creature_night_elf } }
+		}
+		prerequisites = { wc_bank }		
+		gold_cost = 450
+		build_time = 650
+		retinuesize = 100
+		
+		garrison_size = 1
+	}
+	
+	#GoblinDistrict
+	wc_goblinDistrict = {
+		desc = wc_goblinDistrict_desc
+		potential = {
+			always = yes
+			NOT = { owner = { trait = creature_human } }
+			NOT = { owner = { trait = creature_high_elf } }
+			NOT = { owner = { trait = creature_blood_elf } }
+			NOT = { owner = { trait = creature_night_elf } }
+		}
+		upgrades_from = wc_goblinGhetto		
+		gold_cost = 750
+		build_time = 750
+		retinuesize = 150
+		garrison_size = 2
+	}
+	
+		
+	#RestingLounge
+	wc_restingLounge = {
+		desc = wc_restingLounge_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_night_elf }
+			owner = { trait = creature_high_elf }
+			owner = { trait = creature_blood_elf }
+		}
+		gold_cost = 600
+		build_time = 500
+		local_tax_modifier = 0.5
+		garrison_size = 1
+		monthly_character_piety = 0.5
+		ai_creation_factor = 100
+	}
+	
+	#Great Clan Tent
+	wc_greatClanTent = {
+		desc = wc_greatClanTent_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_orc }
+		}
+		gold_cost = 300
+		build_time = 500
+		local_tax_modifier = 0.2
+		monthly_character_piety = 0.3
+		ai_creation_factor = 100
+	}
+	
+	#Kodo Breeder
+	wc_kodoBreeder = {
+		desc = wc_kodoBreeder_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_tauren }
+		}
+		gold_cost = 500
+		build_time = 500
+		trade_route_wealth = 0.2
+		local_movement_speed = 1
+		ai_creation_factor = 100
+	}
+	
+	#MerchantVenue
+	wc_merchantVenue = {
+		desc = wc_merchantVenue_desc
+		potential = {
+			always = yes
+			owner = { trait = creature_human }
+		}
+		prerequisites = { wc_bank }		
+		gold_cost = 350
+		build_time = 650
+		retinuesize = 100
+		garrison_size = 1
+	}
+	
+	#Jewelcrafters
+	wc_jewelcrafters = {
+		desc = wc_jewelcrafters_desc
+		potential = {
+			always = yes
+			port = no
+			OR = {
+			owner = { trait = creature_human }
+			owner = { trait = creature_blood_elf }
+			owner = { trait = creature_goblin }
+			}
+		}	
+		gold_cost = 750
+		build_time = 650
+		tax_income = 0.5
+	}
+	
+	#Quel'dorei Jewelcrafters
+	wc_queldoreijewelcrafters = {
+		desc = wc_queldoreijewelcrafters_desc
+		potential = {
+			always = yes
+			port = no
+			owner = { trait = creature_blood_elf }
+		}	
+		upgrades_from = wc_jewelcrafters		
+		gold_cost = 1000
+		build_time = 1050
+		tax_income = 1
+		monthly_character_prestige = 1
```

Comment:
More than empire title?

Reply:
Ok will nerf that down, any reccomendation?It does give the decision to create a +3 modifier so i need it expensive but how much

Reply:
DONE - 850 gold now